php/WC_Gateway_Blockonomics.php

```diff
@@ -139,7 +139,8 @@ public function process_payment($order_id)
         update_option('blockonomics_orders', $blockonomics_orders);
         $order_url = WC()->api_request_url('WC_Gateway_Blockonomics');
         $order_url = add_query_arg('show_order', $address, $order_url);
-
+        $order_url = str_replace('wc-api/WC_Gateway_Blockonomics', 'crypto', $order_url);
```

Comment:
there is no need to first add WC()->api_request_url and then replace it. We should directly use _show_order_